src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -342,7 +342,7 @@ limitations under the License.
                 <requireCommand>RemoveAllGroups</requireCommand>
                 <requireCommand>AddGroupIfIdentifying</requireCommand>
             </include>
-            <include cluster="Scenes" client="true" server="true" clientLocked="true" serverLocked="true">
+            <include cluster="Scenes" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
Why is this one `clientLocked="true"` while On/Off light was `clientLocked="false"`?

Reply:
What is the semantics of clientLocked? If it is true, the GUI user cannot change whether the client's default?

Reply:
That's a question for @tecimovic ...